tests/pure_executables.list

```diff
@@ -209,6 +209,8 @@ ioctl_hdio-v-Xabbrev
 ioctl_hdio-v-Xraw
 ioctl_hdio-v-Xverbose
 ioctl_inotify
+ioctl_lirc
+ioctl_lirc-success
```

Comment:
*-success executables shouldn't be listed here because they are not pure enough.

Reply:
done